### PR TITLE
refactor: remove deprecated styles and fix scss calc

### PR DESCRIPTION
**Pull request by mcwinter07**
## Why
- deprecation task to remove outdated variables

## What
- replaces $ca-grid with appropriate $spacing token
- update SCSS calculation to use calc and interpolate var
- remove deprecated-component-library-helpers dep

### CODE REVIEW

**Comment by gyfchong** [comment]
Are you missing the removal of `component-library-helpers`?

**Comment by mcwinter07** [comment]
> Are you missing the removal of `component-library-helpers`?

@gyfchong I think this comment might have been from before I amended the commit? if you've got time would love another quick check :)

File: draft-packages/filter-menu-button/KaizenDraft/FilterMenuButton/FilterSplitButton.module.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import "~@kaizen/design-tokens/sass/border";
 @import "~@kaizen/design-tokens/sass/color";
 @import "~@kaizen/design-tokens/sass/typography";
-@import "~@kaizen/deprecated-component-library-helpers/styles/type";
```
Comment (gyfchong) [inline draft-packages/filter-menu-button/KaizenDraft/FilterMenuButton/FilterSplitButton.module.scss:4]: Oh, is this not an actual package install?

Comment (mcwinter07) [reply draft-packages/filter-menu-button/KaizenDraft/FilterMenuButton/FilterSplitButton.module.scss:4]: I believe its being pulled in from component-library, which has a dependency on that

**Review by gyfchong — APPROVED** [review]

**Pull request merged**